addons/website/tests/test_create_website.py

```diff
@@ -0,0 +1,9 @@
+from odoo.tests import HttpCase
+from odoo.tools import mute_logger
+
+class TestCreateWebsite(HttpCase):
+
+    def test_01_create_website(self):
+        with mute_logger('odoo.addons.base.models.ir_translation'):
+            self.env["base.language.install"].create({'lang': 'fr_FR', 'overwrite': True}).lang_install()
```

Comment:
Since you don't really need the translation here (you just need to check that the url is correctly redirect), what you could do is create a new lang to speed up perfs, as that new lang would not have .po files.
`lang_install` is typically taking over 30 seconds to load the files.

See https://github.com/odoo/odoo/commit/2a49a485a6c40185b6ecdf9d74403dd80b873109